lib/components/eureka_segmented_control.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/material.dart';
+import 'package:material_segmented_control/material_segmented_control.dart';
+
+/// This is a custom made Segmented Control made specifically for Project
+/// Eureka. Please use this Widget when implementing a segmented control of
+/// some type to prevent us from repeating ourselves.
+class EurekaSegmenetedControl extends StatefulWidget {
+  final int segmentedControlCount;
+  final List<String> segmentedControlLabels;
+  final int segmentedControlValue;
+  final ValueChanged<dynamic> setState;
+
```

Comment:
consider updating your command to the following 
```
/*
 * comment
 * /
```

Reply:
Disregard this comment, I just cited this document... this is diff than what I'm used to in java and python https://dart.dev/guides/language/effective-dart/documentation